kotest-property/src/jvmTest/kotlin/com/sksamuel/kotest/property/shrinking/FloatShrinkerTest.kt

```diff
@@ -21,7 +20,7 @@ class FloatShrinkerTest : FunSpec() {
          ).map(::row).toTypedArray()
 
          forAll(*values) { value ->
-            FloatShrinker.shrink(value) should beEmpty()
+            FloatShrinker.shrink(value).shouldBeEmpty()
```

Comment:
`should beEmpty()` and `shouldNot beEmpty()` should be supported, it's part of commonly used style, and most assertions do support both `shouldXxx()` and `should Xxx()`. Is there a reason this PR makes `beEmpty` private? I'd advocate for reverting that change, so that this test, and presumably many others in other code bases, can be kept as is.

Reply:
Okay. In that case I'll revert this as unrelated I'll add `beEmpty` matcher factories for all the newly added arrays so that matcher-style and assertion-style are on par.